gql/resolver_maddr.go

```diff
@@ -1,7 +1,8 @@
 package gql
 
 func (r *resolver) MinerAddress() string {
-	return r.provider.Address.String()
+	// TODO: this function doesn;t seem to be used at all. Confirm.
+	return r.provider.Addresses[0].String()
```

Comment:
Not used in Go doesn't necessarily mean not used at all. I believe this is a reference to it from React:

```
const MinerAddressQuery = gql`
    query AppMinerAddressQuery {
        minerAddress
    }
`;
```

Reply:
good spot - thanks!